lsp-metals.el

```diff
@@ -192,15 +193,46 @@ Should be ignored if there is no open doctor window."
     (setq lsp-metals--current-buffer (current-buffer))
     (lsp-notify "metals/didFocusTextDocument" (lsp--buffer-uri))))
 
+(declare-function dap-debug (conf))
+(declare-function dap-register-debug-provider (name conf))
```

Comment:
I think this should be
``` elisp
(declare-function dap-debug "ext:dap-mode" (conf))
(declare-function dap-register-debug-provider "ext:dap-mode" (name conf))
```

Reply:
Yes, you are right - interestingly the byte-compilation tests do not fail...